api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -109,7 +109,7 @@ type VirtualMachineImageStatus struct {
 	Uuid string `json:"uuid,omitempty"` //nolint:revive,stylecheck
 
 	// Deprecated
-	InternalId string `json:"internalId"` //nolint:revive,stylecheck
+	InternalId string `json:"internalId,omitempty"` //nolint:revive,stylecheck
```

Comment:
* Please mark this as deprecated in the correct manner -- https://book.kubebuilder.io/reference/markers/crd.html
* Should also have `// +optional`